doc/nrf/includes/nrf_cloud_rest_sample_requirements.txt

```diff
@@ -6,9 +6,9 @@ This sample uses the `nRF Cloud REST API`_, which requires that your device has
 
 (See `nRF Cloud REST API`_ and `Securely generating credentials on the nRF9160`_ for more details on this requirement).
```

Comment:
Please note once "Securely generating credentials" cloud document is created for nRF9161, we could update this document and link.
I will create a task for this.

Reply:
Updating that page is my next big priority alongside Wi-Fi. Since the scripts we have in the Utils repo also work for the 9161, I'm going to update the page to be 'Securely generating credentials for nRF91 Series devices'.

Reply:
@melwee01 : I can update the link name to "Securely generating credentials for nRF91 Series devices" in link.txt as part of this PR. So, that you need not create a separate PR for this in NCS?